flybywiresim/a32nx
# fix: seatbelt sign sync to fsuipc

Fixes #2752

## Summary of Changes

Integrated the key event that is being sent by the default plane which allows fsuipc (and therefore SLC or PACX) to recognize that

## Screenshots (if necessary)



https://user-images.githubusercontent.com/19493808/111877824-601c2b00-89a5-11eb-800a-16537062f853.mp4


## References





## Additional context



Discord username (if different from GitHub): saschl#9432

## Testing instructions

I don't know another way to test this besides having SLC or PACX.
1. Load in a flight and start the tool (SLC or PACX)
2. Switch the seatbelt signs button to on -> the tool should update accordingly
3. Switch the seatbelt signs button to off -> the tool should update accordingly
4. Perform 2. and 3. but toggle the seatbelt via the external tool. In game state should be updated accordingly


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
